bazel/linux/templates/runner.template.sh

```diff
@@ -115,6 +115,8 @@ while getopts "k:e:r:hsx" opt; do
 done
 shift $((OPTIND - 1))
 
+WRAPPER_OPTS=("$@")
```

Comment:
do we need to add support for wrapper_opts in the vm_bundle as well?

Reply:
Created Jira https://enfabrica.atlassian.net/browse/SF-264